app/src/main/java/com/teamagam/gimelgimel/app/model/entities/LocationEntity.java

```diff
@@ -1,153 +1,58 @@
 package com.teamagam.gimelgimel.app.model.entities;
 
-import android.content.AsyncQueryHandler;
-import android.content.ContentResolver;
-import android.content.ContentValues;
-import android.database.Cursor;
-import android.net.Uri;
+import android.support.annotation.StringDef;
 
 import com.google.gson.annotations.SerializedName;
-import com.teamagam.gimelgimel.app.control.content_provider.GGContentProvider;
-import com.teamagam.gimelgimel.helpers_autodesk.control.content_provider.ProviderEntity;
-import com.teamagam.gimelgimel.helpers_autodesk.control.database.DbEntity;
-import com.teamagam.gimelgimel.helpers_autodesk.control.database.annotations.DbBinder;
-import com.teamagam.gimelgimel.helpers_autodesk.control.database.annotations.DbPrimaryKey;
+import com.teamagam.gimelgimel.app.view.viewer.data.geometries.PointGeometry;
 
-import java.io.Serializable;
-
-import static com.teamagam.gimelgimel.helpers_autodesk.control.content_provider.BaseContentProvider.formUri;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
 
 /**
- * This entity is a "friend" entity from GGMessagingAPI
- * The naming convention is built to enable Gson instance to convert all fields from lower
- * case with underscores to camel case and vice versa.
+ * Created by Gil.Raytan on 10-Jul-16.
  */
-//todo: update all columns and fields and api and...
-public class LocationEntity implements Serializable, DbEntity, ProviderEntity {
-
-    public static final String TABLE_NAME = "GimelGimel";
-
-    public static final Uri CONTENT_URI = formUri(TABLE_NAME, GGContentProvider.authority);
+public class LocationEntity {
```

Comment:
The use of the word entity implies this is a data-base entity associated class. This is not the situation. 
This class holds the combination of geo-location, text and a type, Its name should reflect that.
Perhaps GeoText...
!fix